shimmingtoolbox/shim/realtime_shim.py

```diff
@@ -109,6 +109,7 @@ def realtime_shim(nii_fieldmap, nii_anat, pmu, json_fmap, nii_mask_anat_riro=Non
     for it in range(nt):
         gradient[:][..., it] = phys_gradient(g * fieldmap[:, :, :, it], nii_fieldmap.affine)  # [mT / mm]
     gradient *= 1000  # [mT / m]
+    # gradient *= 1000  # [uT / m]
```

Comment:
Why the same multiplicator for mT and uT ? Do you want to keep the second line ?

Reply:
Uncommenting the second line will output in uT/m, probably not necessary anymore.

Reply:
Resolved in 1a5ce6c.